scale/settings.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python
```

Comment:
While the django style settings.py is ok for now, in the long run I'd prefer we use https://traitlets.readthedocs.io/en/stable/ for configuration - since that's what the rest of the infrastructure (hub, proxy, etc) use.